src/FSharp.Core/option.fsi

```diff
@@ -444,6 +444,7 @@ module Option =
     val inline ofObj: value: 'T -> 'T option  when 'T : null
 #else
     // TODO NULLNESS: assess this change - is it a breaking change?
+    [<WarnOnWithoutNullArgument("You can create 'Some value' directly instead of 'ofObj', or consider not using an option for this value.")>]
```

Comment:
One issue here - string inside the attribute is not localised, whereas diagnostic which will be preceding it will be localised.